omod/src/main/webapp/resources/scripts/visit/visit.js

```diff
@@ -593,6 +594,7 @@ angular.module("visit", [ "filters", "constants", "encounterTypeConfig", "visitS
                                 patient: {
                                     uuid: patientUuid
                                 },
+                                breadcrumbOverride: encodeURIComponent(JSON.stringify(breadcrumbOverride)),
```

Comment:
Not directly related to this commit, but fixes an issue where breadcrumbs weren't working right in the vust note "section" context